pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -561,6 +561,9 @@ func toKubeContainerStatus(status *runtimeapi.ContainerStatus, runtimeName strin
 		State:        toKubeContainerState(status.State),
 		CreatedAt:    time.Unix(0, status.CreatedAt),
 	}
+	if annotatedInfo.Image != "" {
```

Comment:
This will always override the `status.Image.Image` above if the annotated info is present, is that what we want?

Reply:
Before the new label is set, we keep using `status.Image.Image`.
If we can get from the label(info annotation), we will use it instead.
/lgtm